pkg/components/storage.go

```diff
@@ -30,15 +30,13 @@ func startCSIPLugin(kubeconfigPath string) error {
 			"assets/components/odf-lvm/topolvm-csi-resizer_rbac.authorization.k8s.io_v1_clusterrole.yaml",
 			"assets/components/odf-lvm/topolvm-node-scc_rbac.authorization.k8s.io_v1_clusterrole.yaml",
 			"assets/components/odf-lvm/topolvm-node_rbac.authorization.k8s.io_v1_clusterrole.yaml",
-			"assets/components/odf-lvm/topolvm-node-scc_rbac.authorization.k8s.io_v1_clusterrole.yaml",
```

Comment:
Why are we removing the clusterrole and clusterrolebinding for topolvm? Is the new SCC enforcing already the persmissions?

Reply:
This path appeared in the list twice, probably due to an earlier mistake resolving a merge conflict, so it's just cleanup here.